pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
```

Comment:
Although the span calls may themselves be nops, I was worried about the ancillary calls that are used to prepare the parameters for the span calls, like log.Object and fmt.Sprintf. 

I just looked at log.Object and it's a cheap call, so that's fine. But fmt.Sprintf is very expensive and should definitely be hoisted outside of the loops.